webpack.config.publish.js

```diff
@@ -11,33 +13,19 @@ module.exports = baseConfig({
     filename: 'index.js',
     libraryTarget: 'umd'
   },
-  externals: {
-    react: {
-      root: 'React',
-      commonjs2: 'react',
-      commonjs: 'react',
-      amd: 'react',
-      umd: 'react',
-    },
-    'react-dom': {
-      root: 'ReactDOM',
-      commonjs2: 'react-dom',
-      commonjs: 'react-dom',
-      amd: 'react-dom',
-      umd: 'react-dom',
-    },
-    lodash: 'lodash',
-    classnames: 'classnames'
-  },
+  externals: [ nodeExternals() ],
```

Comment:
How's it work if the consumer of this library does not have these `externals`? Is there an obvious error or does yarn know to install as a dependency of this dependency?

Reply:
@PepperTeasdale The package.json + yarn should handle that. It just prevents them from being bundled into index.js